android/quest/src/main/java/org/smartregister/fhircore/quest/ui/patient/details/ListDataDetailViewModel.kt

```diff
@@ -137,4 +137,11 @@ constructor(
     }
     return resourceListLive
   }
+  fun loadQuestionnaireResponse(questionnaireResponseId: String): QuestionnaireResponse? {
+    var questionnaireResponse: QuestionnaireResponse? = null
+    viewModelScope.launch {
```

Comment:
it would be always null? can we either male is suspend and call in launch OR return LiveData and observe value

Reply:
I made it suspend function now.